docs/general/plugin-ca.md

```diff
@@ -0,0 +1,194 @@
+    6N5/96gZ77nUDtbI4qBuHxiGZ0rxcGFJ+/fJTUbKV+QKuF16GRxURUfoyJ5iL9Si
+    AnmwFWYxglgunft9xqW6tg/+0v8J9hcO1uxe3M0LXj4xh5BUCAtOuGaPcE1uHBtI
+    qQ==
+    -----END CERTIFICATE-----
+    -----BEGIN CERTIFICATE-----
+    MIIFFDCCAvygAwIBAgIULu/YsgYLAcQ1kPc8kyzkDEGjhS8wDQYJKoZIhvcNAQEL
+    BQAwIjEOMAwGA1UECgwFSXN0aW8xEDAOBgNVBAMMB1Jvb3QgQ0EwHhcNMjUwNjI0
+    MTIxMTA3WhcNMzUwNjIyMTIxMTA3WjAiMQ4wDAYDVQQKDAVJc3RpbzEQMA4GA1UE
+    AwwHUm9vdCBDQTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBAJaj/7VE
+    AdTGAJoylinnqNzuKKV2ZRV6yqFhMeVknRWl4nGOuJp58sQPO0DXG2uxv1Oi6hKo
+    Q8A2uL3ReQVt60VqrVvoFKFFaBnicnJ9XWOzZWx07uz7PoBc9llj+azUuSrTOWF5
+    wxtQ1RHM/v2fPyzoNQMwj6Xohggh1JboFUW09IRXmoDW/HNVuFdoDtlk47ZAeI7S
+    9z3yHMhTlOJ1tDrQqQgh2booBfm8DhoDtdIkFCjG9kKj9nB2Wz4hM160fneAlg5m
+    aP0TZSECfWq3I0QCadXmveUth6jvU+0TI54O/O6/w/Tm9Sd0VuswoKkxFAH+PgJF
+    /8FifH3BWi0dmLRBSPVBlJiUloFtXeZAsYGjHVlz2hs0R1cL8D0STJwWgLTQGnak
+    CY9j7S/3CwGKMfuCxxDbFDhCcEoFDC4kO6CyU7GXNN8DZhZSBIjXF5Gj1Ua93Co/
+    lmISOxVrFNCEdDODFLEe1dgffUn0m4kWUWaQzbsLWqFQFx1YZs0FjQ61Ap6Y8QjR
+    edhmTGROCZRm9y4HrHRAZJ2poIfXOSJgkyfu/o7kvkO/zhamYKNbmBMJGvlw7JdS
+    waMp4I5kFNql27AAFJVG1lyFGagr7fi7wDsY8ohRB5V/mFV1Hu06Ukz03Z+s5+hj
+    6c2mPxoO5c/hY7QVt8G2gvYkvRpek2iI1IFHAgMBAAGjQjBAMB0GA1UdDgQWBBTB
+    thIKcqmem8YGPAkkqvkUdptflDAPBgNVHRMBAf8EBTADAQH/MA4GA1UdDwEB/wQE
+    AwIC5DANBgkqhkiG9w0BAQsFAAOCAgEAdE95pB1JOlmZkR9WEXb8F81FESti/z2V
+    nKkAQsYui39UK3jK93cMRg2axxLH/3hXxLJcVNZ/iV5aTNhL9naatui3dMz0zLBk
+    2CduGwctlBooJzOa4c2jUbhpdycyIjsHFd6l9ezrWY/JOf1oLwwjNwPa1AO+VOt+
+    ZC4tf4j/O0Q+6ThhGQfZVr0X6UN/jWV89Wpo00QsyACwcn2izbx9o25KSGioNJeS
+    ZcwpgbW1jzASSEUeklqyc1gfZgxM7HyHC+GUV/QSfJugUB4glyUZzpz6gTZWL6N5
+    aq5xkQBSUAP8nOmy4aIAEEx4clL03iq62xbwamzjtET5M5NqRIPc2V2cZqQhs0TJ
+    iiGT98SBu2IydDGPXI/rruujShrIhmJ9WwiaPBdHBnSQQ+AkeDvA3AOcgFmy6Mbs
+    HfJ5vvwxtPYLc8VPNGWKlu+Jbknea+N5izpdSca+TqfqQ+QwVpcbAGgplT5CqmHU
+    Ap0ytVizhMxJpMMDU1GZ2C90SCX9N9hnD/Who/Py1BfbjEvBD9TuNdQ14cRWHDmU
+    Xmyv/zsOhCBskS7bnQNLqhBUS4JMvSDCb0CUMEzmGzJDCGXOTeYs2d1mcNTvDkLS
+    Hgv1jKTfpRXP4pMFOGGMY9XC3OYK/TtVhDAyrWewREMNQTtBKSEj2S6R5rT5MD02
+    ir4ltxRVyHM=
+    -----END CERTIFICATE-----
+    ```
+    > **_NOTE:_** It might be necessary to restart the workload if you only see one certificate.
+1. Update `combined-root.pem` by adding the new root certificate again, this will ensure that workload certificates are regenerated after the next step as the root certificate is updated:
+    ```bash
+    cat root-cert.pem >> combined-root.pem
+    ```
+1. Update `cacerts` secrets to use the new intermediate certificate, key and chain and updated combined root certificates:
+    ```bash
+    kubectl delete secret cacerts -n istio-system --ignore-not-found && \
+    kubectl create secret generic cacerts -n istio-system \
+        --from-file=intermediate/ca-cert.pem \
+        --from-file=intermediate/ca-key.pem \
+        --from-file=root-cert.pem=combined-root.pem \
+        --from-file=intermediate/cert-chain.pem
+    ```
```

Comment:
Why do we delete and recreate `cacerts`? Can we create `cacerts` once with the intermediate?

Reply:
> Problem with doing that in one step is that there will be a time period where some workloads are already updated (using combined root-cert.pem and new intermediate cert) while others are not yet updated and using old root-cert.pem with a single root. 

This is the point of the multiroot setting though I thought? Before updating any workloads to be signed by the new CA, all workloads should already trust both CAs. Then when you update the workloads to be signed by the new CA, old workloads will already trust it.